typescript/widgets/src/components/Tooltip.tsx

```diff
@@ -17,6 +18,7 @@ export function Tooltip({
   size = 16,
   className,
   placement = 'top-start',
+  tooltipClassName = 'max-w-[calc(100%-10px)] sm:max-w-[526px]',
```

Comment:
This is probably okay but generally it's better to have prop classNames append into the default set instead replace it when provided. See clsx